docs/ref/urlresolvers.txt

```diff
@@ -130,6 +130,13 @@ If the URL does not resolve, the function raises a
 
         The name of the URL pattern that matches the URL.
 
+    .. attribute:: ResolverMatch.route
+
+        .. versionadded:: 2.0
```

Comment:
should this be 2.1? or its OK to get into 2.0?

Reply:
Probably 2.1 is fine considering this isn't really related to the new URL feature. It could be mentioned in the release notes.